src/pps/pps.proto

```diff
@@ -815,6 +815,19 @@ message ListPipelineRequest {
 
   // Projects to filter on. Empty list means no filter, so return all pipelines.
   repeated pfs_v2.Project projects = 6;
+
+  // Page indicates which page of a certain size to return. If page is left empty,
+  // all of the selected pipelines will be returned.
+  PipelinePage page = 7; 
```

Comment:
If it’s not too late, I think it might be a good idea to reconsider some of the names here. Is this a pipeline page, or is this a _definition_ or _request_ or _specification_ for a pipeline page?

Reply:
I actually like the name `page` here. It's clear and short. What would you call it?

Reply:
Maybe `PageSpec`? Not a huge deal, I just found an earlier version of the code a little surprising to read. But this right now seems okay:

```go
offset = request.Page.PageIndex * request.Page.PageSize
```